readme.txt

```diff
@@ -37,25 +37,28 @@ Site Kit shows key metrics and insights from different Google products:
 * **Optimize:** Use Site Kit to easily set up Optimize- no code editing required. Then, set up A/B tests in Optimize.
 
 == Installation ==
+**Note**: Make sure that your website is live. If your website isn't live yet, Site Kit can't show you any data. 
+However, if you have a staging environment in addition to your production site, Site Kit can display data from your production site in the staging environment. Learn how to use [Site Kit with a staging environment] (https://sitekit.withgoogle.com/documentation/using-site-kit-on-a-staging-environment/).
 
 = Installation from within WordPress =
 
 1. Visit **Plugins > Add New**.
 2. Search for **Site Kit by Google**.
-3. Install and activate the Site Kit by Google plugin.
+3. Install and activate the Site Kit by Google plugin. If there are multiple WordPress admins, keep in mind that each admin must activate and set up Site Kit in order to access the plugin.
```

Comment:
We might want to clarify this sentence to only include the "each admin must set up...". Only the first admin has to activate the plugin, that's still like WP default behavior. Maybe, to clarify that, we could instead add a 4th step like "Connect Site Kit to your Google account.", and add the extra sentence about other admins there?

Reply:
SGTM on step 4. 
We discussed a bit with Evan about the most appropriate wording in #881. Let's align the language from the start page with this file. 
How about: "Connect Site Kit to your Google account. If there are multiple WordPress admins, keep in mind that each admin must connect their own Google account in order to access the plugin."

Reply:
Sounds good! Fixed, and added this text to the manual installation section.